plugin/pkg/replacer/replacer.go

```diff
@@ -45,6 +47,8 @@ func New(r *dns.Msg, rr *dnstest.Recorder, emptyValue string) Replacer {
 			"{size}":   strconv.Itoa(req.Len()),
 			"{remote}": addrToRFC3986(req.IP()),
 			"{port}":   req.Port(),
+			"{local}":  addrToRFC3986(req.LocalIP()),
+			"{resolver}": getMetadata(ctx, "forward/resolving_proxy"),
```

Comment:
Two things:

1) in the docs, we should note which place holders require the metadata plugin to be enabled
2) the placeholders should match the metadata key

Reply:
the format of labels in metadata is <plugin>/<name>, so it should be easy to check if it 'looks like' a metadata or if it is an error.

Reply:
1. Ok, I added it to log plugin README.md
2. What did you mean? Should we have {resolving_proxy} or {forward/resolving_proxy} placeholders? Or something else?

Reply:
{forward/resolving_proxy}, and if the placholder contains a /, then you getMetaData for it - but as @fturib said in another comment, we probably can't do that last part without a refactor. So for now, just change the placeholder name so it won't change the interface later when we implement that.

Reply:
Ok, I changed the placeholder to be {forward/resolving_proxy}.